CustomApps/lyrics-plus/index.js

```diff
@@ -275,10 +275,13 @@ class LyricsContainer extends react.Component {
             return;
         }
 
-        if (!this.state.synced) return;
+        let lyricsToTranslate = this.state.synced;
+
+        if (!lyricsToTranslate) lyricsToTranslate = this.state.unsynced;
+        if (!lyricsToTranslate) return;
```

Comment:
```suggestion
        let lyricsToTranslate = this.state.synced;

        if (!lyricsToTranslate) {
        	lyricsToTranslate = this.state.unsynced;
        	return;
        }
```
Also is the variable reassigned anywhere? If not then use `const` instead of `let`.

Reply:
well, no it doesn't make sense, it should be
```suggestion
        let lyricsToTranslate = this.state.synced ?? this.state.unsynced;

        if (!lyricsToTranslate) return;
```

Reply:
> well, no it doesn't make sense, it should be

Yea this seems to clear it up a bit!